ember/app/styles/components/share-buttons.scss

```diff
@@ -35,65 +30,40 @@
 
   .share-buttons-wrapper {
     width: $toggle_width + ($border_width * 2);
-
+    border: 1px solid $color_brand-primary;
     overflow: hidden;
     border-radius: 5px;
-
-    border: $border_width solid $color_brand-primary;
-    transition: $transition_border, width .2s;
-
-    &:hover {
-      border-color: $color_brand-primary--active;
-
-      transition: $transition_border, width .2s;
-    }
-
-    &.active {
-      width: $drawer_width + ($border_width * 2.5); 
-
-      transition: width .2s;
-
-      .toggler { background: lighten($color_bg-medium, 5); }
-    }
-  }
-
-  .toggler {
-    float: right;
-    z-index: 1; 
-
-    width: $toggle_width;
-  }
-
-  .share-drawer {
-    position: absolute;
-    top: 0;
-    left: 0;
-    display: flex;
-
-    width: $drawer_width - $toggle_width;
```

Comment:
Optional cleanup: by deleting this, you no longer have need for the `$drawer_width` variable declared at the top of the file scope.